### PR TITLE
[README] FAQ Added.

### PR DESCRIPTION
**Pull request by Fadreus**
Tried to make it simple for anyone to understand.

### CODE REVIEW

**Comment by Nipodemos** [comment]
hello
i just wonder why to put a dot in every title, why?

File: README.md [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
-
-* Before submitting an issue, ensure you are running the latest commit on branch.
-* Check if an issue already exists for your request.
+3. Run openkore.pl _(You can run start.exe or wxstart.exe if you use Windows)_.
+
+## F.A.Q. (Frequently Asked Questions).
+<!-- Source: http://forums.openkore.com/viewtopic.php?f=0&t=11287 -->
+ 1. **Have a problem?**
+    - Update your openkore.
+    - Download anew.
```
Comment (sctnightcore) [inline README.md]: new .

Comment (Fadreus) [reply README.md]: It's literally mean the same but with proper English.

**Comment by Fadreus** [comment]
@Nipodemos To complete a sentence? :no_mouth:
Some are single word, some are in phrase. Having just some title with fullstop doesn't look nice.
Having fullstop on every title make it look uniform.
Or change all title in capital letters? :thinking: 

It just doesn't seems right IMO.

File: README.md (hunk 1 above)
Comment (Nipodemos) [reply README.md]: i though that it was wrong too hahahah
i had to reasearch to know that is correct

Comment (sctnightcore) [reply README.md]: I think use `a new` ?

**Comment by Nipodemos** [comment]
@Fadreus titles are not considered as phrases, so they don't need a dot in the end (i think they even can't have)

i search a few README's on GitHub to see the pattern:
[README of Flutter, a framework made by all mighty Google](https://github.com/flutter/flutter/blob/master/README.md)
[README of VS Code, a code editor made by Microsoft](https://github.com/Microsoft/vscode/blob/master/README.md)
[README of React Native, a framework made by Facebook](https://github.com/facebook/react-native/blob/master/README.md)

in all three repositories, all titles don't have dot

**Comment by Fadreus** [comment]
> titles are not considered as phrases, so they don't need a dot in the end (i think they even can't have)

Fair Enough

Changed as requested. :ok_hand:

**Pull request merged**